chemclipse/plugins/org.eclipse.chemclipse.ux.extension.xxd.ui/src/org/eclipse/chemclipse/ux/extension/xxd/ui/swt/ExtendedWellDataUI.java

```diff
@@ -121,23 +120,6 @@ private void createToolbarMain(Composite parent) {
 		buttonToolbarEdit = createButtonToggleToolbar(composite, toolbarEdit, IMAGE_EDIT, TOOLTIP_EDIT);
 		buttonTableEdit = createButtonToggleEditTable(composite, tableViewer, IMAGE_EDIT_ENTRY);
 		buttonDelete = createButtonDelete(composite);
-		createSettingsButton(composite);
```

Comment:
When changing an element, the size needs to be adjusted if elements are added/removed:
`composite.setLayout(new GridLayout(5, false)); `